eng/ArduinoCsCI.cmd

```diff
@@ -0,0 +1,66 @@
+echo on
```

Comment:
Per my previous comment, let's just remove this file for now. We can later decide to add a separate pipeline that we can conditionally invoke in some PRs that where it is relevant.

Reply:
This is the build script, not the yml file. We might adapt or change this, but deleting won't help.